sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -91,32 +95,88 @@ credential.update("{new_api_key}");
+[FormTrainingAsyncClient][form_training_async_client]. It also supports model management operations to
+get custom model info, delete model, list models and get account details.
+
+### FormRecognizerClient
+A `FormRecognizerClient` and `FormRecognizerAsynClient` provides both synchronous and asynchronous operations to access
+ a specific use of Form Recognizer, such as recognizing layout, receipt and custom form data from documents.
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L39-L42 -->
+```java
+```
+
+### FormTrainingClient
+A `FormTrainingClient` is the Form Recognizer interface to use for creating, using, and managing custom machine-learned models.
+It provides both synchronous and asynchronous operations for training custom models, retrieving and deleting models, 
+as well as understanding how close you are to reaching subscription limits for the number of models you can train.
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L39-L42 -->
+```java
+```
+
+### Long-Running Operations
+Long-running operations are operations which consist of an initial request sent to the service to start an operation,
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
+succeeded, to get the result.
+
+Methods that train models or recognize values from forms are modeled as long-running operations. The client exposes
+a `begin_<method-name>` method that returns a `SyncPoller` or `PollerFlux` instance. 
+Callers should wait for the operation to completed by calling `getFinalResult()` on the returned operation from the
+`begin_<method-name>` method. Sample code snippets are provided to illustrate using long-running operations
+[below](#Examples).
+
+### Training models
+Using the `FormTrainingClient`, you can train a machine-learned model on your own form type. The resulting model will
+be able to recognize values from the types of forms it was trained on.
+
+#### Training without labels
+A model trained without labels uses unsupervised learning to understand the layout and relationships between field
+names and values in your forms. The learning algorithm clusters the training forms by type and learns what fields and
+tables are present in each form type.
+
+This approach doesn't require manual data labeling or intensive coding and maintenance, and we recommend you try this
+method first when training custom models.
+
+#### Training with labels
+A model trained with labels uses supervised learning to recognize values you specify by adding labels to your training forms.
+The learning algorithm uses a label file you provide to learn what fields are found at various locations in the form,
+and learns to recognize just those values.
+
+This approach can result in better-performing models, and those models can work with more complex form structures.
+
+### Recognizing values from forms
```

Comment:
I don't know if it's necessary to break it up with labeled and unlabeled, because we consider that more of an "advanced" sample